src/test/resources/application-test.properties

```diff
@@ -0,0 +1,18 @@
+spring.datasource.url=jdbc:tc:mysql:8.0.24:///lostfilm?useUnicode=true&characterEncoding=UTF-8
+spring.datasource.username=root
+spring.datasource.password=root2021
+logging.level.root =INFO
+spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
+
+spring.quartz.job-store-type=memory
+
+spring.quartz.cron.default = 0/20 0 0 ? * * *
+
+spring.kafka.consumer.group-id=lostfilmTvSeries
+spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
```

Comment:
you have same properties in your application.properties file, place here only properties which you ovveride for your tests